src/storage/storage.cc

```diff
@@ -769,6 +770,98 @@ rocksdb::Status Storage::FlushScripts(engine::Context &ctx, const rocksdb::Write
   return Write(ctx, options, batch->GetWriteBatch());
 }
 
+StatusOr<int> Storage::IngestSST(const std::string &sst_dir, const rocksdb::IngestExternalFileOptions &ingest_options) {
+  std::vector<std::string> sst_files;
+  DIR *dir = opendir(sst_dir.c_str());
```

Comment:
Using the RocksDB API is better:

https://github.com/apache/kvrocks/blob/18e6674eecbfe3361891883f7777c1d188e5ba25/src/storage/storage.cc#L1069

Reply:
Hello @caipengbo the GetChildren does basically the same, why it would be better?

Reply:
1. RocksDB's API is more portable 
2. The overall code style is more unified

Reply:
addressed via 4ac013680a46ec33cdddeb9705b4ab55fb75dde2